app/components/elements/TextLink.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
Let's also remove the `TextLink.js` file, since it's not being used

Reply:
thanks for the catch 🤦‍♂️